tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/core/connection/ConnectionTest.java

```diff
@@ -111,88 +136,122 @@ public void getResourceKindShouldCallClient() {
+		// when
+		// then
+		assertThat(connection).isNotEqualTo(two);
 	}
 
 	@Test
 	public void shouldNotEqualsIfDifferentHost() throws Exception {
+		// given
 		Connection two = new Connection("https://openshift.redhat.com:8443", null, null);
 		two.setUsername("foo");
-
-		assertNotEquals("Exp. connections not to be equal unless they have same url and user", connection, two);
+		// when
+		// then
+		assertThat(connection).isNotEqualTo(two);
 	}
 
 	@Test
 	public void shouldEqualsIfSameUrlAndUser() throws Exception {
+		// given
 		connection.setUsername("foo");
 		Connection two = new Connection("https://localhost:8443", null, null);
 		two.setUsername("foo");
-
-		assertEquals("Exp. connections to be equal if they have same url and user", connection, two);
+		// when
+		// then
+		assertThat(connection).isEqualTo(two);
 	}
 
 	@Test
-	public void cloneShouldCreateIdenticalConnection() throws Exception {
-		// pre-conditions
-		Connection connection = new Connection("https://localhost:8443", null, null);
-		connection.setPassword("foo");
-		connection.setPassword("bar");
-		
-		// operations
-		IConnection clonedConnection = connection.clone();
+	public void shouldEqualIfDifferentPassword() throws Exception {
+		// given
+		connection.setUsername("bar");
+		connection.setPassword("42");
 
-		// verifications
-		assertTrue(clonedConnection instanceof Connection);
-		assertEquals(connection.getUsername(), clonedConnection.getUsername());
-		assertEquals(connection.getPassword(), clonedConnection.getPassword());
-		assertEquals(connection.isRememberPassword(), clonedConnection.isRememberPassword());
-		assertEquals(connection.getHost(), clonedConnection.getHost());
+		Connection two = new Connection("https://localhost:8443", null, null);
+		two.setUsername("bar");
+		two.setPassword("42");
+		// when
+		// then
+		assertThat(connection).isEqualTo(two);
+	}
+	
+	@Test
+	public void shouldEqualIfDifferentToken() throws Exception {
+		// given
+		connection.setUsername("kung");
+		connection.setToken("foo");
+
+		Connection two = new Connection("https://localhost:8443", null, null);
+		two.setUsername("kung");
+		two.setToken("foo");
+		// when
+		// then
+		assertThat(connection).isEqualTo(two);
 	}
 
 	@Test
-	public void shouldNotCredentialsEqualIfDifferentToken() throws Exception {
+	public void should_not_credentials_equal_if_different_username() throws Exception {
+		// given
 		Connection two = (Connection)connection.clone();
-		two.setToken("tokenTwo");
-		assertEquals(connection, two);
+		two.setUsername("userTwo");
+		// when
+		// then
+		assertNotEquals(connection, two);
 		assertFalse(two.credentialsEqual(connection));
 	}
 
 	@Test
-	public void shouldNotCredentialsEqualIfDifferentPassword() throws Exception {
-		Connection two = (Connection)connection.clone();
-		two.setPassword("passwordTwo");
+	public void should_not_credentialsEqual_if_different_password() throws Exception {
```

Comment:
We should either, for consistency, use camelcase or underscore.  I prefer the later because its more readable, but the former is convension